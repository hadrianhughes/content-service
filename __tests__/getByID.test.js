const getByID = require('../lib/getByID');

const mockData = require('../__mock-data__/doc.json');
const mockApi = {
  getByID: string => new Promise ((resolve, reject) => {
    resolve(mockData);
  })
};

describe('getByID function', () => {
  it('Should return a promise which resolves with an object', () => {
    expect(getByID(mockApi, 'foo').resolves).toMatchSnapshot();
  });
});
