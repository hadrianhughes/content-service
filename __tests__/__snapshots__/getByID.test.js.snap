// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getByID function Should return a promise which resolves with an object 1`] = `undefined`;
